### DISCUSSION TITLE
GIT LFS on local repo only?

### DISCUSSION THREAD

**Discussion by jschoeni** [git-lfs/git-lfs#4618] [Q&A]
Hello, where can I find instructions about how to use LFS on a 
local bare repo only? Use case: have text files and connected video
files in one repo, want to separate with LFS.
Only in local repo, no Git Hub involved.

Thanks for help.

**Comment by bk2204** [comment]
As a practical matter, you'll want to have a repository with a working tree to add and remove files.  But you can push to and pull from a local bare repository if you have a sufficiently new version of Git LFS (i.e., 2.10 or newer) and that will work just fine.  I believe 2.10 required the use of a `file:///` URL to do this correctly, and newer versions accept local paths as well.  Do note that currently Windows SMB paths are not supported.

All you need to do is add a remote for the local path or `file:///` URL and push to that remote like normal.

**Reply by jschoeni** [reply]
Thank you,
I tried and get some files listed now when doing
git lfs ls-files in my repo.

Thank you very much for your fast reply.
BR
Jürgen
